content/pt-br/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -1,54 +1,113 @@
+O nome não pode ter mais que 52 caracteres. Esta limitação existe porque o controlador do CronJob adicionará automaticamente 11 caracteres ao final do nome de tarefa escolhido, e o comprimento máximo de um nome de tarefa não pode ultrapassar 63 caracteres.
 
 <!-- body -->
 
-## Limitações do Cron Job
+## CronJob
 
-Um trabalho cron cria um objeto de trabalho _about_ uma vez por tempo de execução de seu planejamento, Dizemos "about" porque há certas circunstâncias em que duas tarefas podem ser criadas ou nenhum trabalho pode ser criado. Tentamos torná-los únicos, mas não os impedimos completamente. Portanto, os trabalhos devem ser _idempotente_.
+CronJobs são úteis para criar tarefas periódicas e recorrentes, como a execução de _backups_ ou o envio de mensagens de e-mail. CronJobs também permitem o agendamento de tarefas individuais para um horário específico, como por exemplo uma tarefa que executa quando a chance do cluster estar ocioso é maior.
 
-Se `startingDeadlineSeconds` estiver definido como um valor grande ou não definido (o padrão) e se `concurrencyPolicy` estiver definido como `Allow(Permitir)` os trabalhos sempre serão executados pelo menos uma vez.
+### Exemplo
 
-Para cada CronJob, o CronJob {{< glossary_tooltip term_id="controller" >}} verifica quantas agendas faltou na duração, desde o último horário agendado até agora. Se houver mais de 100 agendamentos perdidos, ele não iniciará o trabalho e registrará o erro
+Este manifesto de CronJob de exemplo imprime o horário atual e uma mensagem dizendo olá a cada minuto:
+
+{{< codenew file="application/job/cronjob.yaml" >}}
+
+(O artigo [Running Automated Tasks with a CronJob](/docs/tasks/job/automated-tasks-with-cron-jobs/) demonstra este exemplo com maiores detalhes).
+
+### Sintaxe do cronograma cron
+
+```
+# ┌───────────── minuto (0 - 59)
+# │ ┌───────────── hora (0 - 23)
+# │ │ ┌───────────── dia do mês (1 - 31)
+# │ │ │ ┌───────────── mês (1 - 12)
+# │ │ │ │ ┌───────────── dia da semana (0 - 6) (domingo a sábado;
+# │ │ │ │ │                                 7 também representa domingo em alguns sistemas operacionais)
+# │ │ │ │ │
+# │ │ │ │ │
+# * * * * *
+```
+
+| Expressão 								| Descrição																								  	| Equivalente a |
+| ------------- 						| ------------- 																							|-------------  |
+| @yearly (ou @annually)		| Executa uma vez por ano, à meia-noite de 1º de janeiro 			| 0 0 1 1 * 		|
+| @monthly 									| Executa uma vez por mês, à meia-noite do primeiro dia do mês| 0 0 1 * * 		|
+| @weekly 									| Executa uma vez por semana, à meia-noite de domingo					| 0 0 * * 0 		|
+| @daily (ou @midnight)			| Executa uma vez por dia, à meia-noite												| 0 0 * * * 		|
+| @hourly 									| Executa uma vez por hora, no minuto zero         			  		| 0 * * * * 		|
+
+Por exemplo, a linha abaixo determina que a tarefa deve iniciar toda sexta-feira à meia-noite, bem como em todo dia 13 do mês à meia-noite:
```

Comment:
What this lines represents is "a tarefa deve iniciar à meia noite de todos os dias 13 do mês, desde que esse dia seja em uma sexta-feira" :)

Reply:
well, as we discussed in Slack you can disregard this! cron is...confuse :D

Reply:
According to crontab.guru, the schedule for the cron job will be every Friday at midnight, plus every 13th day of the month at midnight too. Would it be clearer if I change the verbiage to this?
> a tarefa deve iniciar à meia-noite de toda sexta-feira e também à meia-noite de todo dia 13 do mês, mesmo que este não ocorra em uma sexta-feira